test/remote/gateways/remote_element_test.rb

```diff
@@ -12,6 +12,25 @@ def setup
       billing_address: address,
       description: 'Store Purchase'
     }
+
+    @google_pay_network_token = network_tokenization_credit_card('4444333322221111',
+      payment_cryptogram: 'EHuWW9PiBkWvqE5juRwDzAUFBAk=',
+      month: '01',
+      year: Time.new.year + 2,
+      source: :google_pay,
+      transaction_id: '123456789',
+      eci: '05')
+
+    @apple_pay_network_token = network_tokenization_credit_card('4895370015293175',
+      month: 10,
+      year: Time.new.year + 2,
+      first_name: 'John',
+      last_name: 'Smith',
+      verification_value: '737',
+      payment_cryptogram: 'abc1234567890',
+      eci: '07',
+      transaction_id: 'abc123',
+      source: :apple_pay)
```

Comment:
💬 Comment:

Is an small silly thing but adding extra data like first_name, last_name and verification_value kind of shows that apple pay tokens somehow are different than google pay tokens.